source/java/spring-boot-web/index.html

```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
-  <title>Spring Boot Stormpath Web Starter &mdash; Spring Boot Stormpath Web Starter 1.0.RC4.5 documentation</title>
+  <title>Spring Boot Stormpath Web Starter &mdash; Spring Boot Stormpath Web Starter 1.0.RC5-SNAPSHOT documentation</title>
```

Comment:
Is this auto-generated? When we release it will not be `-SNAPSHOT` any longer, just `1.0.RC5`. Will it change automatically during the release process?